Formula/g/gcc@13.rb

```diff
@@ -35,6 +36,7 @@ class GccAT13 < Formula
 
   on_linux do
     depends_on "binutils"
+    depends_on "flex"
```

Comment:
Does this use Apple's `flex` on macOS? If so, it should be `uses_from_macos` instead.

Reply:
Modula 2 requires flex. Flex comes with MacOS, but not with the Ubuntu CI container.

Your "instead" puzzles me. Don't you mean "as well"? I would assume that this is the way to go (and that is what I just pushed):

```
  uses_from_macos "flex"

  on_linux do
    ...
    depends_on "flex"
  end
```

Reply:
No, `uses_from_macos` means: "On macOS use the one from system, and on other platforms install this formula"